solidity/test/token_stake/TestStakingPortBacker.js

```diff
@@ -384,6 +384,38 @@ describe("TokenStaking/StakingPortBacker", () => {
       )
     })
 
+    it("allows to copy stake only one time even if it's been recovered", async () => {
```

Comment:
Can we add a case there we call `recoverStake` twice?

Reply:
Not sure if that's what you asked for but it's in https://github.com/keep-network/keep-core/pull/1981/commits/0708dbca4e52105ba169c0a3af6d6bcc76b4e791 😆 

- The test case when `recoverStake` is called twice is already covered for a delegation created from a grant in `TestStakingGrant.js`,
- We had no test case for `recoverStake` called twice for delegation created from liquid tokens - added it in the mentioned commit,
- I don't think we need to add a test case for `recoverStake` in `TestStakingPortBacker` because this contract is not doing anything else than calling `recoverStake` on the staking contract. Did I miss something?